src/gui/sdlmain.cpp

```diff
@@ -1074,26 +1074,21 @@ Bitu GFX_SetSize(Bitu width, Bitu height, Bitu flags,
 			LOG_MSG("SDL:Can't create renderer, falling back to surface");
 			goto dosurface;
 		}
-		/* SDL_PIXELFORMAT_ARGB8888 is possible with most
-		rendering drivers, "opengles" being a notable exception */
-		sdl.texture.texture = SDL_CreateTexture(sdl.renderer, SDL_PIXELFORMAT_ARGB8888,
+		/* Use renderer's default format */
+		SDL_RendererInfo rinfo;
+		SDL_GetRendererInfo(sdl.renderer, &rinfo);
+		const Uint32 textureFormat = rinfo.texture_formats[0];
```

Comment:
Suggest `const uint32_t texture_format = rinfo.texture_formats[0];`

Lowercase_snake format for local variables. 

Unless the API's types are truly unique, we also prefer to just use the standard fixed-types - it's more consistent to read. Alternative is to use `auto` as well.

Reply:
Sorry, I don't think I sent this your way yet :) https://github.com/dosbox-staging/dosbox-staging/blob/master/CONTRIBUTING.md#coding-style

Reply:
oh, good, I like `auto`, done.

Reply:
I'm not seeing snake case further down, though - should I change those as well? e.g.

```
Uint32 pixelFormat;
		SDL_QueryTexture(sdl.texture.texture, &pixelFormat, NULL, NULL, NULL);
```
```
				// does program need to be rebuilt?
				if (sdl.opengl.program_object == 0) {
					GLuint vertexShader, fragmentShader;
```

Reply:
Yes, inheritted code from upstream doesn't follow a consistent convention.. each of their devs has or had a slightly different style. 

But given this is the same variable for the same purpose, then yes - if you'd like to make it consistent in the couple other spots, that would be a nice readability improvement!

Reply:
> Sorry, I don't think I sent this your way yet :) https://github.com/dosbox-staging/dosbox-staging/blob/master/CONTRIBUTING.md#coding-style

Hint taken, thank you!

Reply:
After this, can you squashed these two fixes with the original commit? Then we'll merge.